src/IotHub/IotHub/Common/IotHubDataPlaneUtils.cs

```diff
@@ -60,5 +69,47 @@ public static IEnumerable<PSModules> ToPSModules(IEnumerable<Module> modules)
         {
             return IotHubUtils.ConvertObject<IEnumerable<Module>, IEnumerable<PSModules>>(modules.ToList());
         }
+
+        public static void Validate_Device_Tracing(string DeviceId, string Sku, string Location, bool IsEdgeDevice)
```

Comment:
I don't think the method name follows c# naming convention, should just be PascalCase. Any particular reason?

Reply:
Updated to PascalCase